docs/introduction/welcome.stories.mdx

```diff
@@ -37,7 +37,7 @@ Set up a new app with Circuit UI or add it to an existing project.
 
 File a bug report, suggest a change, or open a pull request.
 
-[Contribute](Introduction/Contributing/Overview)
+[Contribute](https://github.com/sumup-oss/circuit-ui/blob/main/CONTRIBUTING.md)
```

Comment:
Same as above. This page is generated by the `yarn copy-docs` script.

Reply:
done

Reply:
checking the live version, I found those errors, with the following URL, notice the `undefined`

https://circuit.sumup.com/iframe.html?id=features-theme-undefined--page&viewMode=story

Reply:
Where did you find this URL?

Reply:
Found it on [this page](https://circuit.sumup.com/?path=/story/introduction-getting-started--page). Would you mind fixing that, please?